From f53d981abf73de7e5e35eda15b3d9a36e305f885 Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Mon, 3 Mar 2025 11:24:46 +0100
Subject: [PATCH 1653/1695] plat-stm32mp1: drop unused include stm32mp_pm.h

The include file is not needed.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 58ad1f900b27d58d5ff5c35eedae24c231ded5a3 plat-stm32mp1: drop unused include stm32mp_pm.h]
Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: I399486ea83515825c2b83fcd339161977afad787
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/441308
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/plat-stm32mp1/pm/context.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/core/arch/arm/plat-stm32mp1/pm/context.c b/core/arch/arm/plat-stm32mp1/pm/context.c
index 50715cb0e..36e57c0d7 100644
--- a/core/arch/arm/plat-stm32mp1/pm/context.c
+++ b/core/arch/arm/plat-stm32mp1/pm/context.c
@@ -26,7 +26,6 @@
 #include <platform_config.h>
 #include <stdlib.h>
 #include <stm32_util.h>
-#include <stm32mp_pm.h>
 #include <string.h>
 #include <zlib.h>
 
-- 
2.47.2

